components/remote_transmitter.rst

```diff
@@ -48,6 +48,189 @@ Configuration variables:
 -  **id** (*Optional*, :ref:`config-id`): Manually specify
    the ID used for code generation. Use this if you have multiple remote transmitters.
 
+.. _remote-setting-up-infrared:
```

Comment:
I feel like these blocks should be on a page in the `guides` folder and just linked from here.

Reply:
Done.